Library/Formula/shc.rb

```diff
@@ -0,0 +1,17 @@
+class Shc < Formula
+  desc "Shell Script Compiler"
+  homepage "https://neurobin.github.io/shc"
+  url "https://github.com/neurobin/shc/archive/3.9.1.tar.gz"
+  sha256 "7fa6c5c4afe2579cbf92fdf2b71c61504d0b1c0caede3dc1f7e5e95e2c37cb0e"
+
+  def install
+    cd "main" do
+      mkdir_p "#{bin}"
```

Comment:
Is this not done by `make install`? If not, let's go for `bin.mkpath` instead - Looks a bit neater.